lib/src/internal/http/http_route.dart

```diff
@@ -132,6 +135,9 @@ abstract class IHttpRoute {
 
   /// Adds the [`welcome-screen`](https://discord.com/developers/docs/resources/guild#get-guild-welcome-screen) part to this [IHttpRoute].
   void welcomeScreen();
+
+  /// Adds the [`auto-moderation`](https://discord.com/developers/docs/resources/auto-moderation#auto-moderation) part to this [IHttpRoute].
```

Comment:
Should link to https://discord.com/developers/docs/resources/auto-moderation#list-auto-moderation-rules-for-guild